### ISSUE TITLE
cleanup a few sections of uber-development.ini.erb

### ISSUE THREAD

**Issue by binary1230** [magfest/ubersystem-puppet#24]
badge_prices and badge_enums are examples of how to do this right.

interest_list, donation_tier are sloppily coded and we need to replace them with the way that badge_prices and badge_enums are setup.

i.e. it looks like this currently:
```
 donation_tier:
                    - "'No thanks' = 0"
                    - "'Supporter Package' = SUPPORTER_LEVEL"
```

instead if should look something like this:
```
donation_tier:
                    - "No thanks": 0
                    - "Supporter Package": SUPPORTER_LEVEL
```

**Comment by EliAndrewC** [comment]
I agree, but I'm actually planning something a bit broader when it comes to this puppet stuff, FWIW.  I'll ticket it out in advance of doing it so everyone has a chance to digest/react/agree/complain/etc.
